### PR TITLE
Update trigger.md

### PR DESCRIPTION
**Pull request by sarjyusuf**
Changes due to major changes in WF Dashboard Widget and also in the requirement of Service Accounts to run WFs automatially.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/workflows/trigger.md [hunk 1]
```diff
@@ -45,21 +40,21 @@ To trigger a workflow from a Dashboard, add the **Run Workflow** widget:
 1. From your Dashboard, click **Add Widget**.
 1. Search for `workflows` and add the **Run Workflow** widget.
 1. Under **Select the workflow**, find your workflow in the dropdown menu.
-1. Enter a title for the widget and click **Save**
+1. In the second step, you will be asked to map template variables to the workflow input parameters if any.\
+_This will allow the values of the template variables to be mapped directly to the input parameters when you run the Workflow._
+1. Enter a title for the widget if you want and click **Save**
 
 {{< img src="workflows/trigger-from-dashboard.png" alt="Click Run Workflow to trigger a workflow from Dashboard widget." >}}
```
Comment (sarjyusuf) [inline content/en/workflows/trigger.md:47]: @hestonhoffman please use the [image here]([ttps://a.cl.ly/DOukpRr8](https://www.google.com/url?q=https://a.cl.ly/DOukpRr8&sa=D&source=docs&ust=1676046966244913&usg=AOvVaw14a-3TYHenR3QZ6IjPyFY3))

Comment (sarjyusuf) [reply content/en/workflows/trigger.md:47]: [ttps://a.cl.ly/DOukpRr8](https://www.google.com/url?q=https://a.cl.ly/DOukpRr8&sa=D&source=docs&ust=1676046966244913&usg=AOvVaw14a-3TYHenR3QZ6IjPyFY3)

File: content/en/workflows/trigger.md [hunk 2]
```diff
@@ -24,12 +24,7 @@ further_reading:
 
 You can trigger a workflow manually or automatically.
 
-All automatically triggered workflows require you to associate the workflow with a unique service account. Automatic workflow triggers include the following trigger types:
-
-- @mention in a monitor
-- @mention in Security Signal notification rule or detection rule 
-- Scheduled
-
+While you run your Workflow, it can either run with the identity of the user who owns the Workflow or the identity of a service account associated to the Workflow.
```
Comment (hestonhoffman) [inline content/en/workflows/trigger.md:27]:
```suggestion
A workflow can either run with the identity of the user who owns it, or with the identity of a service account associated to the Workflow.
```

File: content/en/workflows/trigger.md [hunk 3]
```diff
@@ -45,21 +40,21 @@ To trigger a workflow from a Dashboard, add the **Run Workflow** widget:
 1. From your Dashboard, click **Add Widget**.
 1. Search for `workflows` and add the **Run Workflow** widget.
 1. Under **Select the workflow**, find your workflow in the dropdown menu.
-1. Enter a title for the widget and click **Save**
+1. In the second step, you will be asked to map template variables to the workflow input parameters if any.\
+_This will allow the values of the template variables to be mapped directly to the input parameters when you run the Workflow._
```
Comment (hestonhoffman) [inline content/en/workflows/trigger.md:44]:
```suggestion
1. Map dashboard template variables to workflow input parameters. This allows the values of your dashboard template variables to be mapped directly to the input parameters when you run the workflow.
```

File: content/en/workflows/trigger.md [hunk 4]
```diff
@@ -45,21 +40,21 @@ To trigger a workflow from a Dashboard, add the **Run Workflow** widget:
...
+1. Enter a title for the widget if you want and click **Save**
```
Comment (hestonhoffman) [inline content/en/workflows/trigger.md:45]:
```suggestion
1. Enter a title for the widget and click **Save**.
```

File: content/en/workflows/trigger.md [hunk 5]
```diff
@@ -45,21 +40,21 @@ To trigger a workflow from a Dashboard, add the **Run Workflow** widget:
...
 
-{{< img src="workflows/trigger-from-dashboard.png" alt="Click Run Workflow to trigger a workflow from Dashboard widget." >}}
+{{< img src="workflows/trigger-from-dashboard2.png" alt="Click Run Workflow to trigger a workflow from Dashboard widget." >}}
 
 To run the workflow:
 1. Click **Run Workflow** on your dashboard widget.
-1. Under **Execution parameters**, enter the required trigger variables for your workflow.
+1. Under **Execution parameters** any values selected in the template variables mapped to the input parameters will automatically be mapped. You can edit these inputs.
```
Comment (hestonhoffman) [inline content/en/workflows/trigger.md:51]:
```suggestion
1. Under **Execution parameters**, any template variables you mapped to workflow inputs are automatically populated. Enter the values for any unmapped execution parameters, or edit the existing the values if needed. 
```

**Review by hestonhoffman — APPROVED** [review]

**Pull request merged**